code/modules/projectiles/projectile/energy.dm

```diff
@@ -20,7 +20,7 @@
 
 /obj/item/projectile/energy/electrode/on_hit(atom/target, blocked = 0)
 	. = ..()
-	if(!ismob(target) || blocked >= 2) //Fully blocked by mob or collided with dense object - burst into sparks!
+	if(!ismob(target) || blocked >= 100) //Fully blocked by mob or collided with dense object - burst into sparks!

```

Comment:
remnants of the old system where blocked could be 0, 1, or 2. blocked is now 0-100.